sbu-kalm/KALM
# sets up flask server with dummy api accessors

**Updates**
1. Implemented flask backend set up
2. Removed express server set up
3. Add dummy api testers

**Testing - Check off with x to verify you've completed the below**

**How to review:**
```
git checkout main
git fetch
git checkout flaskServer
cd server
flask run

In another terminal: 
cd client
npm start
```

1. Go to browser at http://127.0.0.1:5000 to see the below. (The below image just shows a dummy Hello API response)
![image](https://github.com/sbu-kalm/KALM/assets/53535722/ca153b67-833d-488e-bf61-58fb1817868c)

2. Go to localhost:3000 -> Frame Management -> Click on Test Api Button on the top -> You should see below response
![image](https://github.com/sbu-kalm/KALM/assets/53535722/b7367e11-687c-41e7-be0b-9138d2e68f9a)
